rm-ethereum.conf

```diff
@@ -0,0 +1,14 @@
+{
+"gasPrice":"1000000000",
+"maxgasLimit":"1000000000",
+"debugInfo":"false",
+"rpcAddress":"http://localhost:8545",
+"channelAddr":"0x0f1736FA3eB54BD8C234725BD658201DA0B4190D",
```

Comment:
Just wondering why the channel address is specified here? Aren't we creating the channels and getting new ones? Or is this the channel contract address?

Reply:
Yep, it's the Contract address.
I will update it's name, to channelContractAddress.
I also want to all Settings to the library as they are the same on both sides.
We will do it after the merge with David changes.